src/components/InputDatePicker.js

```diff
@@ -16,60 +16,73 @@ const propTypes = {
   identifier: PropTypes.string,
   showTimePicker: PropTypes.bool,
   viewMode: PropTypes.string,
+  sideBySide: PropTypes.bool,
   disabled: PropTypes.bool,
-  daysOfWeekDisabled: PropTypes.array
+  daysOfWeekDisabled: PropTypes.array,
 };
 
 const defaultProps = {
-  startDate: '',
-  className: 'datepicker--input',
-  viewMode: 'days',
+  startDate: "",
+  className: "datepicker--input",
+  viewMode: "days",
+  sideBySide: true,
 };
 
 export function openAtDate(input, date) {
-  setProperty($(input), 'viewDate', date);
+  setProperty($(input), "viewDate", date);
   input.focus();
 }
 
 function setProperty($picker, property, value) {
-  return $picker.data('DateTimePicker')[property](value);
+  return $picker.data("DateTimePicker")[property](value);
 }
 
 export default class InputDatePicker extends React.Component {
   componentDidMount() {
-    const { showTimePicker, startDate, viewMode, minDate, maxDate, daysOfWeekDisabled } = this.props;
+    const {
+      showTimePicker,
+      startDate,
+      viewMode,
+      sideBySide,
+      minDate,
+      maxDate,
+      daysOfWeekDisabled,
+    } = this.props;
     const format = showTimePicker ? YEAR_MONTH_DAY_TIME : YEAR_MONTH_DAY;
     const $picker = $(this.picker).datetimepicker({
       format,
       defaultDate: startDate,
       useCurrent: false, // If we have no initial value, we keep the input blank
       keyBinds: null,
-      sideBySide: true,
+      sideBySide: sideBySide,
```

Comment:
Can use object shorthand syntax for properties that have the same name: 

const $picker = $(this.picker).datetimepicker({
  format,
  defaultDate: startDate,
  useCurrent: false
  keyBinds: null,
  sideBySide,
  viewMode,
});

Reply:
Great point. Thanks KG!